crates/sparrow-merge/src/gather/gathered_batches.rs

```diff
@@ -0,0 +1,22 @@
+use sparrow_batch::{Batch, RowTime};
+
+/// The result of a gather operation
+#[derive(Debug, PartialEq)]
+pub struct GatheredBatches {
```

Comment:
Does this really merit a separate file? It seems like it's "the result of gather" so it could go in `gather.rs` near the top as part of the API?

Reply:
Eh, maybe. I made this in the assumption that I'd move some more stuff into `gather`, but since I haven't I'll move this over and we can split it in the future if we need to.